packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5744,6 +5758,38 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   };
 
+  /**
+   * Internal API to return status of BNR
+   * @returns {Boolean}
+   * @public
+   * @memberof Meeting
+   */
+  isBnrEnabled() {
+    return this.bnrStatus === BNR_STATUS.ENABLED;
+  }
+
+  /**
+   * Internal API to obtain BNR enabled MediaStream
+   * @returns {Promise<MediaStreamTrack>}
+   * @private
+   * @param {MedaiStreamTrack} audioTrack from updateAudio
+   * @memberof Meeting
+   */
+  async internal_enableBNR(audioTrack) {
+    try {
+      LoggerProxy.logger.info('Meeting:index#internal_enableBNR. Internal enable BNR called');
+      const bnrAudioTrack = await WebRTCMedia.Effects.BNR.enableBNR(audioTrack);
+
+      LoggerProxy.logger.info('Meeting:index#internal_enableBNR. BNR enabled track obtained from WebRTC & returned as stream');
+
+      return bnrAudioTrack;
+    }
+    catch (error) {
+      LoggerProxy.logger.error('Meeting:index#internal_enableBNR.', error);
+      throw error;
+    }
+  }
+
   /**
    * enableBNR API
    * @returns {Promise<Boolean>}
```

Comment:
I guess we're returning a Boolean value `isSuccess` not a Promise<Boolean>. Correct me if I'm missing something.